dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeFactoryImpl.java

```diff
@@ -683,4 +683,21 @@ public void validateFields(ContentType type) {
     
   }
 
+ @Override
+ public void updateModDate(ContentType type) throws DotDataException {
+	 ContentTypeBuilder builder =
+		        ContentTypeBuilder.builder(type).modDate(DateUtils.round(new Date(), Calendar.SECOND));
+	 type = builder.build();
```

Comment:
you can see similar logic here:

https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeFactoryImpl.java#L325-L326